packages/terra-date-time-picker/tests/nightwatch/components/DateTimePickerExcludeDates.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import DateTimePickerExampleTemplate from './DateTimePickerExampleTemplate';
+
+const DateTimePickerExample = () => (
+  <DateTimePickerExampleTemplate
+    excludeDates={['2017-08-14']}
```

Comment:
Does this component exclude or include specific time selection in addition to dates?

Reply:
These are used only in the date picker.